test/manifest-v7.14.js

```diff
@@ -236,7 +236,6 @@ function getExpectedVector(version) {
           {
             type: 'id',
             id: 'wikidata',
-            values: [],
```

Comment:
The `values` property is not expected anymore for field definitions with `skipCopy` property set to `true`.